docs/UserGuide.md

```diff
@@ -117,41 +116,64 @@ Examples:
 
 ### Listing assignments : `list`
 
-Format: `list [NUMBER]`
+Format: `list [NUMBER_OF_DAYS]`
+
+You can list all your assignments with `list`. Alternatively, you can type `list` followed by an index `NUMBER_OF_DAYS` to list your assignments with deadlines that fall within the current date (and time) and `NUMBER_OF_DAYS` later (in number of hours).
+
+The `NUMBER_OF_DAYS` in hours is multiplied by 24.
+
+For example, `list 3` lists all your assignments that are due within 3 days (72 hours) from the current date (and current time). If the current date and time is 24/10/2020 12:00 pm, all assignments due from this date and time to 27/10/2020 12:00PM will be displayed.
+
+DIAGRAM
 
-- Shows a list of assignments in your schedule within next `NUMBER` 
-days, starting from the current date (and current time).
-- A day represents 24 hours. 
-- `list` without `NUMBER` displays your entire list of assignments 
-stored in ProductiveNUS.
+<div markdown="span" class="alert alert-success">
+**:bulb: Tip:**
+You can use this `NUMBER_OF_DAYS` index to quickly view assignments that you need to complete soon!
 
+</div>
+
+More examples: 
+- `list`
+- `list 7`
```

Comment:
Okay, this might be redundant but perhaps adding "lists all your assignments" and "lists all your upcoming assignments in the coming week" or something similar might be a nice touch!